package.json

```diff
@@ -5,9 +5,9 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "start": "nodemon index.js",
+    "start-dev": "NODE_ENV=development nodemon index.js",
     "startdb": "node database/database_setup.js",
-    "test": "test"
+    "env-setup": "touch .env & echo 'AIRTABLE_KEY=' >.env"
```

Comment:
this is the script that creates a new file  (`touch .env`) and then creates some initial content inside of that file (`echo 'AIRTABLE_KEY=' >.env`). let me know if you need more explanation!